testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/Suite2.java

```diff
@@ -15,7 +15,7 @@
 
```

Comment:
Please take a look at product tests README.md if there are no references to this env.